src/components/Error/Error.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Error.css';
+import { gsap, Power3 } from 'gsap';
+
+const Error = () => {
+
+  return (
+    <div className='error-container'>
+      <img
+        alt='Professor Oak'
+        className='prof-oak'
+        src='Images/prof-oak.png'
+      ></img>
+      <div className='error-message'>
+        <h2 className='error-quote'>"Everybody makes a wrong turn once in a while!"</h2>
+        <p className='error-note'>Try again. Enter a real Generation 1 Pokemon name or ID (1 to 152).</p>
```

Comment:
Our app only has 1 -151, we'll need to change this text

Reply:
I think this is a very easy change! Would you mind changing that before we merge?

Reply:
Thanks, I'll do that. :)